### PR TITLE
updates for tests, classes

### PR DESCRIPTION
**Pull request by scottcreynolds**
handles #8 and #9 AND targets the right branch for merge!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "type": "git",
+    "url": "git+https://github.com/learn-co-curriculum/space-invaders.git"
+  },
+  "keywords": [
+    "js",
+    "classes",
+    "oop"
+  ],
+  "author": "scottcreynolds",
+  "license": "ISC",
```
Comment (pletcher) [inline package.json]: I might have missed this in other repos (I can update as needed, no need to go back), but the value of "license" should be "SEE LICENSE IN LICENSE.md".

Also, could we add an "engines" field?

```
"engines": {
    "node": "6.x",
    "npm": "3.x"
  },
```

Students'll need Node.js >= 6 :)

File: requires.yml [hunk 2]
```diff
@@ -1,7 +0,0 @@
-# A list of javascript files required for the project, in dependency order
-javascripts:
-  files:
-    - lib/crewMember.js
-    - lib/spaceship.js
-  specs:
-    - spec/space-invaders-spec.js
```
Comment (pletcher) [inline requires.yml]: 🎉 👯 🎉

**Comment by pletcher** [comment]
I left a couple of comments on the package.json -- apologies if I've missed those in other labs -- just had a student run into some legacy stuff and want to try to head it off with that `"engines"` field :)

Otherwise, this is soooo much better than before!

**Comment by scottcreynolds** [comment]
@pletcher okay cool I'll make sure I update those in the future. changes made on both branches

**Pull request merged**